plugins/chat/spec/jobs/chat_channel_delete_spec.rb

```diff
@@ -68,11 +68,16 @@
                                   }.by(-1).and change {
                                           ChatUpload.where(chat_message_id: @message_ids).count
                                         }.by(-10).and change {
-                                                ChatMessage.where(id: @message_ids).count
-                                              }.by(-20).and change {
-                                                      ChatMessageReaction.where(
-                                                        chat_message_id: @message_ids,
-                                                      ).count
-                                                    }.by(-10)
+                                                UploadReference.where(
+                                                  target_id: @message_ids,
+                                                  target_type: "ChatMessage",
+                                                ).count
+                                              }.by(-10).and change {
+                                                      ChatMessage.where(id: @message_ids).count
+                                                    }.by(-20).and change {
+                                                            ChatMessageReaction.where(
+                                                              chat_message_id: @message_ids,
+                                                            ).count
+                                                          }.by(-10)
   end
```

Comment:
about this formatting here, try to add parenthesis.

Reply:
I'm not sure what you mean. Anyway I'm really not a fan of this, I might rewrite it to just count the records before and then count them again after rather than having this huge >>>>>>>>>>>> nest.